Mathlib/Topology/Algebra/Valued/LocallyCompact.lean

```diff
@@ -148,4 +151,185 @@ lemma totallyBounded_iff_finite_residueField [IsDiscreteValuationRing 𝒪[K]] :
 
 end FiniteResidueField
 
+section CompactDVR
+
+open Valued
+
+variable (K) in
+lemma exists_norm_coe_lt : ∃ x : 𝒪[K], 0 < ‖(x : K)‖ ∧ ‖(x : K)‖ < 1 := by
```

Comment:
This name is very brief for what it's proving. Maybe `exists_norm_coe_lt_one` is better, and similarly for the next two?